src/main/java/frc/robot/swerve/SwerveConfig.java

```diff
@@ -58,48 +49,32 @@ public class SwerveConfig {
     public static final int BR_TURN_CANID = 20;
     public static final boolean BR_REVERSE = false;
 
-    // helper function
-
-
-
-
-
-
-    ///////////////////////////////////////////// UNDERCONSTRUCTION
-
-
-
-
-    //The createDriveMotor method 
-
-    public static void createDriveMotor(int canld, boolean reverseMotor){
-
-        TalonFX DriveMotor = new TalonFX(canld);
-        {
```

Comment:
An opening brace creates a new code block - you don't generally do it unless you're using it for control flow (i.e. the body of a method, or an if statement, or for loop, or something like that. And you don't need to do it here.